### ISSUE TITLE
A connection to the server could not be established.

### ISSUE THREAD

**Issue by OzzyForMe** [Tustin/PlayStationDiscord#199]
**Information**
- OS: Windows
- Version: Latest

**Describe the bug**

**Log**
I cannot log in with the browser in-app, is there anyway of fixing it? Every time I try to log-in, I get either timed-out or the connection ws not estblished.

**Comment by ProLiteAD** [comment]
Just had this problem yesterday, tried again the today and it worked for me.

This probably won't fix it but I pressed on the "Can't sign in" button the went back to the normal sign in page and it worked. I have 2FA enabled and my PS was not on when I signed in on PlayStationDiscord App.

**Comment by kiehnle** [comment]
Confirmed. happening to me on 3.2. I downgraded to 3.0.2, and that worked, but wont show me online.

**Comment by kiehnle** [comment]
Here's my workaround. I downloaded the patch from https://github.com/inthecatsdreams/PlayStationDiscord/releases/tag/v3.2.0-1. Then reinstalled 3.2.0. I was still signed on after the update.

**Comment by Izofeu** [comment]
Logging in in 3.1.2 works just fine, along with displaying the correct status. Simply refuse the admin permissions every time you exit the app and it won't update.

**Comment by diegolamanno** [comment]
Just in case folks land in here like myself. 

I was experiencing similar issues with `3.2` and I needed that version to support PS5. The problem in `3.2` seems to be related to the new Sony oauth flow introduced in this version. 

While going trough the codebase and testing things out. I notice the following
- When I pulled the repo (as of Jan 3 2022) the `package.json` shows version `3.3`
- While running the app locally (in development mode) I was able to get it to work.

So, I am not sure if either: 
- @Tustin fixed something in the last couple of days (I have not compared the commits since `3.2`) and it has not been released yet as a version
- Or the problem is when electron builds the executable.

**If you want to get it to work. Maybe just run it locally.** 

I will try to get an executable from this and see if that works too! 

Cheers,
